examples/codestral-code-interpreter-js/README.md

```diff
@@ -0,0 +1,56 @@
+# AI Code Execution with Mistral's Codestral
+
+This project tests the capabilities of the new Mistral's model on data analysis tasks, using the [Code Interpreter SDK](https://github.com/e2b-dev/code-interpreter) by E2B. Codestral doesn't support using tools yet, so in this Python example, we added the code interpreting capabilities.
+
+We are going to build an AI assistant that performs data analysis on a CSV file. It is prompted to plot the average temperature over the years in Algeria,
+
+
+# Installation
+
+First, cone the E2B Cookbook repository, and navigate to the correct directory.
+
+
+## Install dependencies
+
+Ensure all dependencies are installed via `npm install`.
+
+
+## Set up environment variables
+
+Create a `.env` file in the project root directory and add your API keys:
+
+- Copy `.env.template`   to `.env`
+- Get the [E2B API KEY](https://e2b.dev/docs/getting-started/api-key)
+- Get the [MISTRAL API KEY](https://console.mistral.ai/api-keys/)
+
+# Usage
+
+Ensure you have the [`city_temperature.csv`](https://github.com/e2b-dev/e2b-cookbook/blob/main/examples/codestral-code-interpreter-js/city_temperature.csv) dataset file in the project root directory.
```

Comment:
They will have it there if they cloned the repo. So not sure if you need to mention it